RecoTauTag/Configuration/python/RecoTauTag_EventContent_cff.py

```diff
@@ -112,7 +113,9 @@
         'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationNeutralIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationPUcorrPtSum_*_*',
-        'keep *_hpsPFTauTransverseImpactParameters_*_*'
+        'keep *_hpsPFTauTransverseImpactParameters_*_*',
```

Comment:
Next time I'll try to better pick the line to comment so that the context is displayed better.